lib/exabgp/configuration/neighbor/__init__.py

```diff
@@ -171,6 +175,9 @@ def post (self):
 		neighbor.operational      = capability.get('operational',False)
 		neighbor.route_refresh    = capability.get('route-refresh',0)
 
+		if neighbor.peer_address is None:
+			return self.error.set('incomplete neighbor, missing peer address')
+
```

Comment:
here we should check if we got the IP address from the name and if so assign it, it provide backward compatibility and allow the new behaviour ..